build.rs

```diff
@@ -2,17 +2,44 @@ use cmake::Config;
 use std::{env, path::PathBuf};
 
 fn main() {
-    let dst = Config::new("libllvm-pdb-wrapper").build();
-    println!("cargo:rustc-link-search=native={}", dst.display());
-    println!("cargo:rustc-link-lib=static=llvm-pdb-wrapper");
+    println!("Remember to set the LIBCLANG_PATH and/or LLVM_DIR if needed");
+    if std::env::var_os("CARGO_FEATURE_LLVM_13").is_some() {
+        println!("specialbuold");
```

Comment:
debug print statement left in unintentionally to signify that the build script correctly entered the build code for llvm13